index.html

```diff
@@ -77,8 +98,48 @@ <h3 class="project_name">Project name goes here</h3>
           </button>
         </div>
       </article>
-      <article class="project">
-        <img class="project_img" src="img/project4.png" alt="project cover" />
+      <article class="project_img_2">
+        <img
+          class="project_image_img"
+          src="img/project_img_2.svg"
+          alt="decoration image"
+        />
+      </article>
+      <article class="project_img_1">
```

Comment:
Making the website responsive is the main focus of this project, on a tablet this project would have a black right half of the page because of how you were forcing the elements on the left.

Reply:
Addressed 👌 Great job at following ReshyResh suggestion and making the necessary changes to avoid hard-coded values that prevent the website from being responsive!